### PR TITLE
fix: updated pagination tests to only test order along with forward a…

### PR DESCRIPTION
**Pull request by JoblersTune**
…nd backward pagination

## Changes proposed in this pull request

- Update for explicit testing of forward and backward pagination with ascending and descending ordering. 

## Checklist

- [ ] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [ ] Documentation added
- [x] Make sure that all checks pass
- [ ] Postman collection updated

### CODE REVIEW

**Comment by sabineschaller** [comment]
Does this now only test one sort order?

**Comment by JoblersTune** [comment]
@sabineschaller it tests ascending and descending for forward pagination and backward pagination but only in this one shared test:
```
test.each`
      order             | description
      ${SortOrder.Asc}  | ${'Backwards/Forwards pagination results in same order for ASC.'}
      ${SortOrder.Desc} | ${'Backwards/Forwards pagination results in same order for DESC.'}
    `('$description', async ({ order }): Promise<void> => {
      if (order === SortOrder.Desc) {
        modelsCreated.reverse()
      }
      const paginationForwards = {
        first: 10
      }
      const modelsForwards = await getPage(paginationForwards, order)
      const paginationBackwards = {
        last: 10,
        before: modelsCreated[10].id
      }
      const modelsBackwards = await getPage(paginationBackwards, order)
      expect(modelsForwards).toHaveLength(10)
      expect(modelsBackwards).toHaveLength(10)
      expect(modelsForwards).toEqual(modelsBackwards)
    })
```

File: packages/backend/src/shared/baseModel.test.ts [hunk 1]
```diff
@@ -18,9 +18,6 @@ export const getPageTests = <Type extends BaseModel>({
       for (let i = 0; i < 22; i++) {
         modelsCreated.push(await createModel())
       }
-      if (sortOrder === SortOrder.Desc) {
```
Comment (sabineschaller) [inline packages/backend/src/shared/baseModel.test.ts:21]: Line 13 still picks a sort order and random, doesn't it?

Comment (JoblersTune) [reply packages/backend/src/shared/baseModel.test.ts:21]: I had moved this logic down but I actually think Max wanted it out as well. So I've removed here.

File: packages/backend/src/shared/baseModel.test.ts [hunk 2]
```diff
@@ -53,21 +50,27 @@ export const getPageTests = <Type extends BaseModel>({
...
+      await expect(getPage(pagination)).rejects.toThrow(expectedError)
     })
 
-    test('Backwards/Forwards pagination results in same order.', async (): Promise<void> => {
+    test.each`
+      order             | description
+      ${SortOrder.Asc}  | ${'Backwards/Forwards pagination results in same order for ASC.'}
+      ${SortOrder.Desc} | ${'Backwards/Forwards pagination results in same order for DESC.'}
+    `('$description', async ({ order }): Promise<void> => {
+      // model is in DESC order
```
Comment (sabineschaller) [inline packages/backend/src/shared/baseModel.test.ts:61]: Do we still need this comment?

**Review by sabineschaller — DISMISSED** [review]
LGTM now

**Review by mkurapov — APPROVED** [review]

**Comment by sabineschaller** [comment]
You need to fix your linting issues. Otherwise, I think we are good to go.

**Review by sabineschaller — APPROVED** [review]
Thank you!

**Pull request merged**